ocean-client/src/programs/vault-maxi-program.ts

```diff
@@ -36,42 +59,123 @@ export class VaultMaxiProgram extends CommonProgram {
                 transaction == VaultMaxiProgramTransaction.TakeLoan
     }
 
-    nextCollateralValue(vault: LoanVaultActive) : number {
-        let nextCollateral= 0
-        vault.collateralAmounts.forEach(collateral => {
-            if( collateral.symbol == "DUSD") {
-                nextCollateral += Number(collateral.amount) * 0.99 //no oracle price for DUSD, fixed 0.99
-            } else {
-                nextCollateral += Number(collateral.activePrice?.next?.amount ?? 0) * Number(collateral.amount)
-            }
-        })
-        return nextCollateral
-    }
-
     
-    nextLoanValue(vault: LoanVaultActive) : number {
-        let nextLoan = 0
-        vault.loanAmounts.forEach(loan => {
-            if( loan.symbol == "DUSD") {
-                nextLoan += Number(loan.amount) // no oracle for DUSD
-            } else {
-                nextLoan += Number(loan.activePrice?.next?.amount ?? 1) * Number(loan.amount)
+    async doValidationChecks(telegram:Telegram) : Promise<boolean> {
+        if(!super.doValidationChecks(telegram)) {
+            return false
+        }
+        const vaultcheck = await this.getVault()
+        if(!vaultcheck || +vaultcheck.loanScheme.minColRatio >= this.settings.minCollateralRatio) {
+            const message= "Could not find vault or minCollateralRatio is too low. "
+            + "trying vault " + this.settings.vault + " in " + this.settings.address + ". "
+            + "thresholds " + this.settings.minCollateralRatio + " - " + this.settings.maxCollateralRatio + ". loanscheme minim is " + vaultcheck.loanScheme.minColRatio
```

Comment:
"loanscheme minim is"
would either prefer
- "loanscheme min is"

or

- "loanscheme minimum is"